src/libraries/sendtohelixhelp.proj

```diff
@@ -294,7 +294,7 @@
         <TestTarget>$(AppleTestTarget)</TestTarget>
       </XHarnessAppBundleToTest>
       <!-- Create work items for run-only apps -->
-      <XHarnessAppBundleToTest Condition="Exists('$(TestArchiveRoot)runonly')" Include="$([System.IO.Directory]::GetDirectories('$(TestArchiveRoot)runonly', '*.app', System.IO.SearchOption.AllDirectories))" >
+      <XHarnessAppBundleToTest Condition="Exists('$(TestArchiveRoot)runonly') and '$(TargetOS)' != 'tvOS'" Include="$([System.IO.Directory]::GetDirectories('$(TestArchiveRoot)runonly', '*.app', System.IO.SearchOption.AllDirectories))" >
```

Comment:
this should work on tvOS right? please file an issue so we follow up to reenable the "run-only" apps

Reply:
The reference I have for runonly not working on tvOS is something @steveisok said in chat - Steve, is there an actual record of "runonly doesn't work on tvOS"?

Reply:
We can't get a return code from iOS or tvOS device, which the functional tests expect.

Not sure if there's an issue in xharness. @premun would know.

Reply:
If that were the case then this would fail on iOS too (and it doesn't seem to).

Reply:
@akoeplinger have you tested on device? Right now we have no Apple device runs anywhere

Reply:
Ah I thought we ran it on the staging pipeline? anyway, if it doesn't work on iOS we should exclude it too :)

Reply:
No `iOS_` runs in runtime-staging.yml.

It likely doesn't work on iOS either, but it'll take me ages to check on device via CI.

I could exclude it too, here, though. Or exclude neither since we don't know the state of things for sure.

Reply:
I can test this for you on an iPhone. Can you give me instructions on how to build the app?